processor/src/advice/mod.rs

```diff
@@ -103,7 +98,7 @@ pub trait AdviceProvider {
     ///
     /// # Errors
     /// Returns an error if the key is already present in the advice map.
-    fn insert_into_map(&mut self, key: Word, values: Vec<Felt>) -> Result<(), ExecutionError>;
+    fn insert_into_map(&mut self, key: Word, map: Vec<Felt>) -> Result<(), ExecutionError>;
```

Comment:
It should remain `values` here since this vector defines values which are to be inserted at the specified key.

Reply:
True, was a rebase mistake. Fixed!